ethereum/spec/src/main/java/tech/pegasys/teku/spec/executionlayer/ExecutionLayerChannelStub.java

```diff
@@ -392,6 +425,10 @@ public SafeFuture<HeaderWithFallbackData> builderGetHeader(
   public SafeFuture<BuilderPayload> builderGetPayload(
       final SignedBeaconBlock signedBeaconBlock,
       final Function<UInt64, Optional<ExecutionPayloadResult>> getCachedPayloadResultFunction) {
+    if (!online) {
+      return SafeFuture.failedFuture(new RuntimeException("stub is offline"));
```

Comment:
Can probably extract this in a method cause too many repetitions

Reply:
mm not super sure how we can effectively do that

Reply:
done